includes/digital-twins-limits.md

```diff
@@ -15,8 +15,8 @@ The table below lists the functional limits of Azure Digital Twins.
 | --- | --- | --- | --- |
 | Azure resource | Number of Azure Digital Twins instances in a region, per subscription | 10 | Yes |
 | Digital twins | Number of twins in an Azure Digital Twins instance | 200,000 | Yes |
-| Digital twins | Number of incoming relationships to a single twin | 5,000 | No |
-| Digital twins | Number of outgoing relationships from a single twin | 5,000 | No |
+| Digital twins | Number of incoming relationships to a single twin. See [Modeling Best Practices](../articles/digital-twins/concepts-models.md#best-practices-for-designing-models) for alternatives. | 5,000 | No |
```

Comment:
Bailey I'm wondering to keep the table concise should this be a "note" on these two items which is expanded below?  Just a stylistic comment.

Reply:
See Update.